### DISCUSSION TITLE
cant run anything

### DISCUSSION THREAD

**Discussion by lgwjames** [bmaltais/kohya_ss#2711] [Q&A]
followed this guide

https://sd-scripts.readthedocs.io/en/latest/install.html

but no start bat or gui?

**Comment by mistermcluvin** [comment]
Hi, I think you are looking for Kohya SS GUI vs Kohya SD Scripts?
Looking at the link in your post, you see this?
git clone https://github.com/kohya-ss/sd-scripts.git
vs
git clone --recursive https://github.com/bmaltais/kohya_ss.git

Please go to https://github.com/bmaltais/kohya_ss and follow the install steps from there if you are looking for the GUI version.
